src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -11638,6 +11638,14 @@ public void CompilingCodeWithInvalidPreProcessorSymbolsShouldProvideDiagnostics(
                 Diagnostic(ErrorCode.WRN_DefineIdentifierRequired).WithArguments("1").WithLocation(1, 1));
         }
 
+        [Fact]
+        public void WhitespaceInDefine()
```

Comment:
>WhitespaceInDefine [](http://example.com/codeflow?start=20&length=18)

Would it make sense to test `MemoryExtensions.TrimStart()` directly?

Reply:
Consider testing `TrimStart()` with a string that contains only spaces.

Reply:
I considered direct testing, but it's not what we commonly do for internal APIs. We typically verify bugs via their impact on public APIs.

Added a test with only spaces.

For `define:` with an empty string, that is already covered by test `InvalidDefineSwitch`.